test/parallel/test-stream-pipeline.js

```diff
@@ -477,3 +477,17 @@ const { promisify } = require('util');
     { code: 'ERR_INVALID_CALLBACK' }
   );
 }
+
+{
+  const r = new Readable();
```

Comment:
I know the rest of the test likely is missing these, but would you mind including a code comment in here about what this block is specifically testing for?

Reply:
Added